articles/azure-app-configuration/concept-private-endpoint.md

```diff
@@ -55,7 +55,7 @@ When you create a private endpoint, the DNS CNAME resource record for the config
 
```

Comment:
Can we move the clarification to the end of the paragraph above since it's about the DNS records? Suggest below:

Enabling geo-replication creates separate DNS records for each replica with unique IP addresses in the private DNS zone.

Reply:
Updated

Reply:
I mean the paragraph **above**, not below.

Reply:
Updated